src/structures/gdt.rs

```diff
@@ -105,6 +105,38 @@ impl GlobalDescriptorTable {
         }
     }
 
+    /// Forms a GDT from a slice of `u64`.
+    ///
+    /// # Safety
+    ///
+    /// * The user must make sure that the entries are well formed
+    /// * The provided slice **must not be larger than 8 items** (only up to the first 8 will be observed.)
+    #[inline]
+    #[cfg(feature = "nightly")]
```

Comment:
The `nightly` feature is a collection of a number of smaller-scope features. Here, the `const_fn` feature would make most sense:

```suggestion
    #[cfg(feature = "const_fn")]
```